src/Native/Bootstrap/platform.unix.cpp

```diff
@@ -70,4 +70,24 @@ extern "C"
     {
         throw "RhCompatibleReentrantWaitAny";
     }
+
+    void EnumDynamicTimeZoneInformation()
```

Comment:
Before this change, the tests have been running against stale System.Private.CoreLib from the CLI drop. I have fixed it in this change. These stubs are needed by live System.Private.CoreLib.